### PR TITLE
Small improvements of displayed strings on login screen

### PR DESCRIPTION
**Pull request by zetok**
Messages that user faces should be a bit more clear now.

### CODE REVIEW

File: src/widget/loginscreen.cpp [hunk 1]
```diff
@@ -82,19 +82,19 @@ void LoginScreen::onCreateNewProfile()
...
     {
-        QMessageBox::critical(this, tr("Couldn't create a new profile"), tr("The passwords are different."));
+        QMessageBox::critical(this, tr("Couldn't create a new profile"), tr("The passwords you've entered are different.\nPlease make sure to enter same password twice."));
         return;
     }
 
     if (Profile::profileExists(name))
     {
-        QMessageBox::critical(this, tr("Couldn't create a new profile"), tr("This profile already exists."));
+        QMessageBox::critical(this, tr("Couldn't create a new profile"), tr("Profile with this name already exists."));
```
Comment (tux3) [inline src/widget/loginscreen.cpp]: > Profile with this name already exists.

Should probably be "A profile with this name already exists."

Comment (SkyzohKey) [inline src/widget/loginscreen.cpp]: "A profile with this name already exists." is nice

Comment (ProMcTagonist) [inline src/widget/loginscreen.cpp]: What @tux3 said.

**Comment by zetok** [comment]
Corrected

**Pull request merged**